mod-feesfines/src/main/resources/domain/mod-feesfines/features/automateFeeFines.feature

```diff
@@ -0,0 +1,228 @@
+Feature: automate fee/fines
+
+  Background:
+    * url baseUrl
+    * callonce login testUser
+    * configure headers = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/json, text/plain' }
+    * def mockLoanId = call uuid1
+    * def userId = call uuid1
+    * def groupId = call uuid1
+    * def feefineId = call uuid1
+    * def ownerId = call uuid1
+    * def accountId = call uuid1
+    * def instanceTypeId = call uuid1
+    * def contributorNameTypeId = call uuid1
+    * def instanceId = call uuid1
+    * def holdingId = call uuid1
+    * def servicePointId = call uuid1
+    * def locationId = call uuid1
+    * def institutionId = call uuid1
+    * def campusId = call uuid1
+    * def libraryId = call uuid1
+    * def itemId = call uuid1
+    * def permanentLoanTypeId = call uuid1
+    * def materialTypeId = call uuid1
+    * def checkOutByBarcodeId = call uuid1
+    * def overdueFinePoliciesId = call uuid1
+    * def lostItemFeePolicyId = call uuid1
+    * def loanPolicyId = call uuid1
+    * def patronPolicyId = call uuid1
+    * def requestPolicyId = call uuid1
+
+  Scenario: verify Account.contributors field
+    # instance and its prerequisites
+    * def requestInstanceTypeEntity = read('samples/instance-type-request-entity.json')
+    Given path 'instance-types'
+    And request requestInstanceTypeEntity
+    When method POST
+    Then status 201
+
+    * def requestContributorNameTypeEntity = read('samples/instance-type-request-entity.json')
+    Given path 'contributor-name-types'
+    And request requestContributorNameTypeEntity
+    When method POST
+    Then status 201
```

Comment:
Do we need both?

Reply:
we need just contributor Id, we may mock it and  omit creating this entity, but I've already faced tons of situation when mock Id doesn't work for some complicated computing, so I suggest stay it as now.